wsgicli.py

```diff
@@ -202,14 +204,20 @@ def import_from_path(import_path):
 def find_modules_from_path(import_path):
     import_from_path(import_path)
 
+    def parent_path(path):
+        return os.path.abspath(os.path.join(path, ".."))
+
+    site_dirs = site.getsitepackages()
+    lib_dirs = [parent_path(path) for path in site_dirs]
+
     for module in sys.modules.values():
         path = getattr(module, '__file__', '')
         if path[-4:] in ('.pyo', '.pyc'):
             path = path[:-1]
         if path and os.path.exists(path):
             # Standard libraries are in lib/python3.x/...
```

Comment:
Thanks to your nice Pull Request, this comments are no longer needed.
Please remove this lines.